proto/pbft.proto

```diff
@@ -34,6 +35,24 @@ message pbft_msg
 
     // for checkpoints
     string state_hash = 6;
+
+    // for join/leave requests
+    pbft_peer_info peer_info = 7;
+
+    // for viewchange
+    // TODO replace string with wrapped_bzn_message
+    repeated string checkpoint_messages = 8;
```

Comment:
bzn_envelope

Reply:
Whups, missed this one. Changed string to ban_envelope... Fixing the fallout.